### PR TITLE
Optimize the code

### PR DESCRIPTION
**Pull request by wohaopa**

### CODE REVIEW

**Comment by wohaopa** [comment]
Don't worry, these commits are forward compatible. For example: https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/470

**Comment by wohaopa** [comment]
Now, I use the same coding rules as for PR 470

**Comment by Dream-Master** [comment]
@wohaopa can you rebase your pr against kubas branch because i get merge conflicts whn try to build a dev test version

**Comment by wohaopa** [comment]
Now, this PR has only been optimized for code. Maybe I will open a new PR in the future to complete other work I wanted to do before.

File: src/main/java/appeng/client/gui/implementations/GuiAmount.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public void initGui() {
+        super.initGui();
+
+        final int a = AEConfig.instance.craftItemsByStackAmounts(0);
+        final int b = AEConfig.instance.craftItemsByStackAmounts(1);
+        final int c = AEConfig.instance.craftItemsByStackAmounts(2);
+        final int d = AEConfig.instance.craftItemsByStackAmounts(3);
```
Comment (OneEyeMaker) [inline src/main/java/appeng/client/gui/implementations/GuiAmount.java:55]: Please, make sure that correct config is used and make this base class compatible with https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/192 (to allow changing config).

File: src/main/java/appeng/client/gui/implementations/GuiPatternMulti.java [hunk 2]
```diff
@@ -139,115 +76,65 @@ public void drawFG(final int offsetX, final int offsetY, final int mouseX, final
...
-            } else {
-                resultI = (int) ArithHelper.round(resultD, 0);
-            }
-            out = Integer.toString(resultI);
-            this.symbolSwitch.set(resultI >= 0 ? ActionItems.MULTIPLY : ActionItems.DIVIDE);
-            this.amountToSet.setText(out);
+        if (btn == this.symbolSwitch) {
+            int resultI = -getAmount();
+
+            // this.symbolSwitch.set(resultI >= 0 ? ActionItems.MULTIPLY : ActionItems.DIVIDE);
```
Comment (OneEyeMaker) [inline src/main/java/appeng/client/gui/implementations/GuiPatternMulti.java:113]: Please, remove all commented code to prevent future confusions.

**Review by OneEyeMaker — CHANGES_REQUESTED** [review]
Nice job!
There are few nitpicks I want to be addressed.
Also it would be great if you continue this project in AE2FC.

**Comment by Dream-Master** [comment]
@wohaopa now you need to run spotless

**Review by OneEyeMaker — APPROVED** [review]

**Pull request merged**